user-guide.html

```diff
@@ -417,6 +424,110 @@
 
             <hr/>
 
+            <h2><a id="kotlin"></a>The Kotlin module</h2>
+            <p>
+                Since version 2.1.0, a separate Kotlin module is also available, providing an even nicer DSL to create your database setups.
+                This module is a very thin layer on top of the Java API, so all we discussed in this guide applies to Kotlin, too. The
+                type-safe builders, lambda expressions and extension functions that Kotlin provides allow writing your database setup
+                in a nice, idiomatic, and extensible way.
+            </p>
+            <p>Here is an example of how you would use the Kotlin DSL to write a setup using Kotlin:</p>
+            <pre class="prettyprint">
+    val testDestination = DataSourceDestination(testDataSource)
+
+    val setup = dbSetup {
+        destination = testDestination                   // mandatory
+        binderConfiguration = customBinderConfiguration // optional
+        deleteAllFrom("PRODUCT", "VENDOR")
+        insertInto("VENDOR") {
+            columns("ID", "CODE", "NAME")
+            values(1L, "AMA", "Amazon")
+            values(2L, "PMI", "Prime Minister")
+        },
+        insertInto("PRODUCT")
+            mappedValues("ID" to 1L, 
+                         "NAME" to "Kindle", 
+                         "VENDOR_ID" to 1L)             // using column/value pairs
+            mappedValues("ID" to 2L, 
+                         "NAME" to "Television", 
+                         "VENDOR_ID" to 2L)       
+            withGeneratedValue("FIRST_SELLING_DATE", ValueGenerators.dateSequence())
+        }
+    }</pre>
+
+            <p>
+                You can launch of course launch the setup immediately:
+            </p>
+            <pre class="prettyprint">
+    dbSetup {
+        ...
+    }.launch()</pre>
+
+            <p>or you can launch it with a DbSetupTracker:</p>
+            <pre class="prettyprint">
+    dbSetup {
+        ...
+    }.launchWithTracker(tracker)</pre>

```

Comment:
I'm wondering if I shouldn't name this method `launchIfNecessary(tracker)`, just like the DbSetupTracker.launchIfNecessary(). I also wonder if I should add this methof directly on the DbSetup (in the Java code), rather than as an extension method. It could be handy in Java too.